plugins/BEdita/Core/src/Model/Behavior/DataCleanupBehavior.php

```diff
@@ -57,17 +57,34 @@ public function beforeMarshal(Event $event, \ArrayObject $data)
         if (!empty($data['id'])) {
             return;
         }
-        $config = $this->getConfig();
-        $key = Inflector::underscore($this->_table->getAlias());
+
+        $type = Inflector::underscore($this->_table->getAlias());
         if ($this->_table->behaviors()->has('ObjectType') && $this->_table->objectType()) {
-             $key = $this->_table->objectType()->get('name');
+             $type = $this->_table->objectType()->get('name');
         }
-        $defaults = Configure::read(sprintf('DefaultValues.%s', $key), []);
-        $config['fields'] = array_merge($config['fields'], $defaults);
-        foreach ($config['fields'] as $key => $value) {
+        $fields = $this->defaultFields($type);
+        foreach ($fields as $key => $value) {
             if (!isset($data[$key]) || $data[$key] === null || $data[$key] === '') {
                 $data[$key] = $value;
             }
         }
     }
+
+    /**
+     * Retrieve default fields reading from configuration.
+     *
+     * @param string $type Type name.
+     * @return array
+     */
+    protected function defaultFields(string $type): array
+    {
+        $fields = (array)$this->getConfig('fields');
+        // set default `on` if minimum level is `on`
+        if (Configure::read('Status.level') === 'on') {
+            $fields['status'] = 'on';
+        }
```

Comment:
I don't think it's necessary. This is just to avoid the use of `draft` global default if no default value is specified in `DefaultValues` configuration.